### DISCUSSION TITLE
How do I change the minumum for the string length in binary ninja?

### DISCUSSION THREAD

**Discussion by 0x62797465** [Vector35/binaryninja-api#4679] [Q&A] [answered]
![image](https://github.com/Vector35/binaryninja-api/assets/104478197/ba49eac2-6aa6-416b-823a-f108861287bf)
How do I make it so that binary ninja comments strings of smaller lengths?
I've also changed this and it didn't work
![image](https://github.com/Vector35/binaryninja-api/assets/104478197/9b805629-6ac8-4d57-babd-af1a70177af7)

**Comment by psifertex** [comment]
Did you reanalyze the file after changing that setting? What file format is this? Elf? Macho? macOS type libraries unfortunately don't yet have libc support that would also include the type information for printf that would also influence the rendering of that data as a string. If you double click the data reference I assume you see the strings? Finally, is this with the latest version of Binary Ninja?

**Reply by 0x62797465** [reply]
It’s the latest dev release as of this morning, it’s an elf binary compiled with gcc (without any optimization), I set this option a whole back

**Reply by psifertex** [reply]
Can you share the exact binary? Just zip and you should be able to add it in a comment here. Thanks!

**Reply by xusheng6** [reply]
I already got a repro on this. Debugged it a while and got lost and forgot to follow up. I can post a binary as well

**Reply by xusheng6** [reply, marked as answer]
I have created an issue to track it: https://github.com/Vector35/binaryninja-api/issues/4697. The binary can be found there

**Comment by 0x62797465** [comment]
Fixed, thanks!
